### PR TITLE
Feature/reduce push steps

### PR DESCRIPTION
**Pull request by krlosMata**

### CODE REVIEW

**Review by ignasirv — APPROVED** [review]
💯

File: main/precompiled/pre-ecMul.zkasm [hunk 1]
```diff
@@ -1,18 +1,18 @@
...
  *  - stack input: [x1, y1, s]
  *  - stack output: [x, y]
  */
 funcEcMul:
 
-    %MAX_CNT_BINARY - CNT_BINARY - 4000     :JMPN(outOfCountersBinary)
+    %MAX_CNT_BINARY - CNT_BINARY - 20000     :JMPN(outOfCountersBinary)
     %MAX_CNT_ARITH - CNT_ARITH - 20000      :JMPN(outOfCountersArith)
-    %MAX_CNT_STEPS - STEP - 130000          :JMPN(outOfCountersStep)
+    %MAX_CNT_STEPS - STEP - 175000          :JMPN(outOfCountersStep)
```
Comment (hecmas) [inline main/precompiled/pre-ecMul.zkasm:15]: The worst case is 172.970. Don't you prefer, just in case, to go to 180k?

**Review by hecmas — APPROVED** [review]
🎖️

File: main/precompiled/pre-ecMul.zkasm (hunk 1 above)
Comment (krlosMata) [reply main/precompiled/pre-ecMul.zkasm:15]: I think 175K is enough given the 5% safe guard that we are applying

**Pull request merged**